app/views/users/sessions/new.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="row">
```

Comment:
May I suggest changing the hash rockets (`=>`) for ruby19 hash syntax (that is `as: resource_name`)?